public/v4/apps/shiori.yml

```diff
@@ -1,40 +1,73 @@
 captainVersion: 4
-
 services:
+    # Shiori
     $$cap_appname:
-        image: radhifadlillah/shiori@sha256:3d1eb1f0aeb9804dea41f3ec5d29de5cf214bb66ef04dce1aa942ca17caec7bb
+        image: ghcr.io/go-shiori/shiori:$$cap_shiori_version
         environment:
-            SHIORI_DIR: /data
+            SHIORI_DBMS: mysql
+            SHIORI_MYSQL_USER: $$cap_mariadb-user
+            SHIORI_MYSQL_PASS: $$cap_mariadb-pass
+            SHIORI_MYSQL_NAME: $$cap_mariadb-db
+            SHIORI_MYSQL_ADDRESS: tcp(srv-captain--$$cap_appname-mariadb)
         volumes:
-            - $$cap_appname-shiori-data:/data
-        restart: on-failure
+            - $$cap_appname-data:/shiori
+        restart: unless-stopped
```

Comment:
jfyi - this property is not being used by CapRover and it's ignored. Docker swarm automatically takes care of this.